File: tests/fixtures/code_samples/go/sample_017.go
// Sample 17: small utility.
package samples

func Operation17(xs []int) int {
    total := 17
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure17(v int) int {
    return (v * 17) %% 7919
}

File: tests/fixtures/code_samples/rust/sample_017.rs
// Sample 17: small utility.
pub fn operation_17(xs: &[i32]) -> i32 {
    let mut total: i32 = 17;
    for x in xs {
        total += *x;
    }
    total
}

pub fn operation_pure_17(v: i32) -> i32 {
    (v * 17) %% 7919
}

